File: space-game/WebApi/Classes/Camera.vb
Imports Camera

Namespace Classes
    Public Class Camera

        Private ReadOnly _cameraService As ICameraService

        Public Sub New(cameraService As ICameraService)
            _cameraService = cameraService
        End Sub

        Public ReadOnly Property CurrentTarget As Guid
            Get
                Return _cameraService.CurrentTarget
            End Get
        End Property
    End Class
End NameSpace

File: space-game/WebApi/Classes/Camera.cs
using Camera;
using System;

namespace WebApi.Classes
{
    public class Camera
    {
        private readonly ICameraService _cameraService;
        public Camera(ICameraService cameraService)
        {
            _cameraService = cameraService;
        }

        public Guid CurrentTarget => _cameraService.CurrentTarget;
    }
}
